ParkBlazerApp/src/pages/UserReports.tsx

```diff
@@ -0,0 +1,66 @@
+import { IonPage, IonGrid, IonRow, IonCol, IonCard, IonCardHeader, IonCardTitle, IonCardContent, IonList, IonItem, IonThumbnail, IonLabel, IonText, IonContent } from "@ionic/react";
+import Navbar from "../components/navbar";
+import { useState } from "react";
+import { useEffect } from "react";
+import axios from "axios";
+import AuthService from "../utils/AuthService";
+import "./UserReports.css";
+import { formatDistanceToNow } from "date-fns";
+import { de } from "date-fns/locale";
+
+const UserReports: React.FC = () => {
+    const [reports, setReports] = useState<any>();
+
+    useEffect(() => {
+        fetchAvailabilityReports();
+    }, []);
+
+    const fetchAvailabilityReports = async () => {
+        const token = AuthService.getToken();
+        try {
+            const response = await axios.get('https://server-y2mz.onrender.com/api/user_reports', {
+                headers: {
+                    Authorization: `Bearer ${token}`
+                }
+            });
+            const reportsData = response.data;
+            setReports(reportsData.reports);
+            console.log(reports)
+        } catch (error) {
+            console.error("Fehler beim Abrufen der Parkplatzverfügbarkeitsberichte:", error);
+        }
+    };
+
+
+    return (
+        <IonPage>
+            <Navbar />
+            <IonContent>
+                <IonGrid fixed className="userreports-grid">
+                    <IonRow className="ion-justify-content-center ion-align-items-center full-height">
+                        <IonCol className="userreports-col" size="12" size-sm="12" size-md="12">
+                            <IonText color="primary" className="register-title">
+                                <h1 className="userreports-heading">Deine Meldungen</h1>
+                            </IonText>
+                            {reports && reports.map((report: any, index: any) => (
+                                <IonCard key={index}>
+                                    <IonCardContent>
+                                        <IonText><strong>Datum:</strong> {report.report_date} {formatDistanceToNow(report.report_date, { addSuffix: true, locale: de })}</IonText>
```

Comment:
Mit ``format`` kann man das Datum schöner darstellten:
```suggestion
                                        <IonText><strong>Datum:</strong> {format(report.report_date, "dd.MM.yyyy hh:mm")} Uhr {formatDistanceToNow(report.report_date, { addSuffix: true, locale: de })}</IonText>
```